src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java

```diff
@@ -1,12 +1,83 @@
 package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
 
+import lombok.RequiredArgsConstructor;
+import org.hl7.fhir.dstu3.model.DateType;
+import org.hl7.fhir.dstu3.model.Enumerations;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Identifier;
 import org.hl7.fhir.dstu3.model.Patient;
 import org.springframework.stereotype.Component;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.PersonName;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup.InvestigationSubject;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup.PatientDetails;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup.ServiceReportDetails;
+import uk.nhs.digital.nhsconnect.lab.results.utils.UUIDGenerator;
+
+import java.util.Optional;
 
 @Component
+@RequiredArgsConstructor
 public class PatientMapper {
+
+    protected static final String NHS_NUMBER_SYSTEM = "https://fhir.nhs.uk/Id/nhs-number";
+    private final UUIDGenerator uuidGenerator;
+
     public Patient mapToPatient(final Message message) {
```

Comment:
I believe we're missing the address mapping `ADR++US` (`UnstructuredAddress.java`) in the `PatientMapper`